tests/postgres_tests/test_indexes.py

```diff
@@ -79,7 +79,13 @@ def test_deconstruction(self):
         path, args, kwargs = index.deconstruct()
         self.assertEqual(path, 'django.contrib.postgres.indexes.GinIndex')
         self.assertEqual(args, ())
-        self.assertEqual(kwargs, {'fields': ['title'], 'name': 'test_title_gin'})
+        self.assertEqual(kwargs, {'fields': ['title'], 'name': 'test_title_gin',
+                                  'fastupdate': None, 'gin_pending_list_limit': None})
+
+    def test_illegal_list_limit(self):
+        with self.assertRaisesMessage(ValueError, 'gin_pending_list_limit must be between {} and {}'.format(
```

Comment:
```
msg = '....'
with self.assertRaisesMessage(ValueError, msg):
```
(I think hardcoding the values in the message rather than using format() would be simpler -- is there a reason they would change?)